tests/src/test/java/alluxio/client/cli/fsadmin/command/UpdateConfIntegrationTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.cli.fsadmin.command;
+
+import alluxio.cli.fsadmin.FileSystemAdminShell;
+import alluxio.cli.fsadmin.command.PathConfCommand;
+import alluxio.cli.fsadmin.command.UpdateConfCommand;
+import alluxio.client.cli.fsadmin.AbstractFsAdminShellTest;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests for updateConf command.
+ */
+public final class UpdateConfIntegrationTest extends AbstractFsAdminShellTest {
+
+  @Test
+  public void unknownOption() {
+    int ret = mFsAdminShell.run("updateConf", "--unknown");
+    Assert.assertEquals(-1, ret);
+    String output = mOutput.toString();
+    Assert.assertEquals(UpdateConfCommand.description(), lastLine(output));
+  }
+
+  @Test
+  public void unknownKey() {
+    int ret = mFsAdminShell.run("updateConf", "--option", "unknown-key=unknown-value");
+    Assert.assertEquals(-1, ret);
+  }
+
+  @Test
+  public void updateNormal() {
+    int ret = mFsAdminShell.run("updateConf", "--option", "alluxio.master.worker.timeout=4min");
+    Assert.assertEquals(0, ret);
```

Comment:
run 
`mFsShell.run("getConf", "alluxio.master.worker.timeout");` and assert the returned value?

Reply:
`getConf ` is not a subCommand of `FsShell`, so how to run it through a shell?

Reply:
you are right